src/standalone/api/kernels/index.ts

```diff
@@ -109,7 +106,6 @@ export function getKernelsApi(extensionId: string): Kernels {
                                 // it is effectively preventing access to it.
                                 const deferrable = createDeferredFromPromise(Promise.resolve(thenable));
                                 waitUntil(thenable);
-                                const disposable = progress.show();
```

Comment:
No need to display this, this will happen today automatically when extension attempts to